build/yamls/externalnode/vm-agent-rbac.yml

```diff
@@ -0,0 +1,101 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: vm-agent
+  namespace: test-ns # Change the namespace to where vm-agent is expected to run.
```

Comment:
If we will have a document to run VM agent with the sample yamls, I suggest to use a name that can be used in the installation guide without changing the yaml, e.g. "vm-ns".

Reply:
namespace -> Namespace